author/topics/collections/configuration.adoc

```diff
@@ -171,6 +171,10 @@ coverpage and boilerplate) [added in https://github.com/metanorma/metanorma/rele
 This is automatically included when `sectionsplit` is set in the
 Metanorma file, to break a single document up into multiple HTML files.
 
+`flavor`:::
```

Comment:
In the document we call a flavor `:mn-document-class: iso`. Can we use a single nomenclature for this?

Reply:
We should accept the same names both in the collection config and as document attribute. We could either call it "document-class" or accept both "flavor" and "document-class".

Reply:
I'll go with the latter; document-class is, in retrospect, too ambiguous to recommend.

Reply:
I agree. And maybe we should get rid of the "mn-" prefix one day too...

Reply:
I'm going to merge this one first, do the document attribute rename of "document-class" to "flavor" in a separate PR.